client/src/Components/DashboardSidebar.js

```diff
@@ -44,31 +44,31 @@ export default class DashboardSidebar extends Component {
             visible={visible}
             width='thin'
           >
-            <Menu.Item as='a'>
-              <Link to="/login">
-                <Icon name='home' />
-                Home
-              </Link>
+            <Menu.Item>
+              WELCOME USER!
```

Comment:
Ideally would have the user's name. I'd prefer not to use all caps.

Reply:
fixed as Welcome User